js/URL.js

```diff
@@ -1,17 +1,21 @@
-import { state } from './globalState.js';
+import { state, settings } from './globalState.js';
 import { profileToMatrix } from './utils.js';
 import { loadMatrix } from './InstanceManagement.js';
 import { buildTable } from './TableBuilder.js';
 
 function writeURL() {
-    let stateString = `${state.committeeSize}&` + profileToMatrix(state).replaceAll("\n", "&").slice(0, -1);
+    let profileMatrix = settings.useWeights? "1&" : "0&" 
```

Comment:
I think that this change of format makes all existing URLs invalid, and it would be better to be backwards compatible. I'd rather deduce whether weights are used from whether `*` is used or not, rather than passing a flag.